components/infobox/commons/infobox_person.lua

```diff
@@ -49,6 +49,13 @@ function Person:createInfobox()
 		error('You need to specify an "id"')
 	end
 
+	-- check if non-representing is used and set an according value in self
+	-- so it can be accessed in the /Custom modules
+	args.country = Person:getStandardNationalityValue(args.country or args.nationality)
+	if args.country ~= Person:getStandardNationalityValue('non-representing') then
+		self.notNonRepresenting = true
```

Comment:
This it makes more sense to have it the other way around

Reply:
do you mean this?
```lua
	if args.country == Person:getStandardNationalityValue('non-representing') then
		self.nonRepresenting = true
```

alternatively could also name it `self.isRepresenting`

Reply:
Yeah that's what I meant